frontend/src/metabase/components/Link.jsx

```diff
@@ -4,17 +4,20 @@ import PropTypes from "prop-types";
 import { Link as ReactRouterLink } from "react-router";
 import styled from "styled-components";
 import { display, color, hover, space } from "styled-system";
+
 import { stripLayoutProps } from "metabase/lib/utils";
+import Tooltip from "metabase/components/Tooltip";
 
 BaseLink.propTypes = {
   to: PropTypes.string.isRequired,
   disabled: PropTypes.bool,
   className: PropTypes.string,
   children: PropTypes.node,
+  tooltip: PropTypes.string,
```

Comment:
One of the downsides of the `forwardRefToInnerRef` util I added is that you can't wrap it again in a `styled(...)`, as is the case here with the `Link` component. Instead, I'm adding a `tooltip` prop to skip the refs.